include/net/aws_iot.h

```diff
@@ -18,84 +19,69 @@
 /**
  * @defgroup aws_iot AWS IoT library
  * @{
- * @brief Library to connect the device to the AWS IoT message broker.
+ * @brief Library used to connect an IP enabled device to the AWS IoT Core MQTT broker.
  */
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
-/** @brief AWS IoT shadow topics, used in messages to specify which shadow
- *         topic that will be published to.
+/** @brief AWS IoT shadow topic types.
+ *	   Used to address messages to the AWS IoT shadow service.
  */
 enum aws_iot_shadow_topic_type {
 	/** Unused default value. */
 	AWS_IOT_SHADOW_TOPIC_NONE,
+
 	/** This topic type corresponds to
-	 *  $aws/things/<thing-name>/shadow/get, publishing an empty message
+	 *  $aws/things/<thing-name>/shadow/get, publishing an empty string
```

Comment:
I interpret `empty string` in C context to be simply a null-terminator, ie a message with a pointer to `\0` and size of `1`. Is that correct in this case?

Reply:
yes